core/src/ledger/masp_conversions.rs

```diff
@@ -454,6 +459,19 @@ where
     // obtained
     wl_storage.storage.conversion_state.tree =
         FrozenCommitmentTree::merge(&tree_parts);
+    // Update the anchor in storage
+    let anchor_key = Key::from(MASP.to_db_key())
+        .push(&MASP_CONVERT_ANCHOR_KEY.to_owned())
+        .expect("Cannot obtain a storage key");
+    wl_storage.write(
+        &anchor_key,
+        crate::types::hash::Hash(
```

Comment:
We can abstract all of this out into a `hash` function on the tree type, perhaps?

Reply:
Ah the `FrozenCommitmentTree` is imported from the masp crate but we can probably refactor this part of the code in other ways